packages/runner/src/lib/event-manager.js

```diff
@@ -237,6 +237,14 @@ const eventManager = {
       studioRecorder.startSave()
     })
 
+    reporterBus.on('studio:copy:to:clipboard', (cb) => {
```

Comment:
As as a heads up, I moved `event-manager` to a new shared package, so it can be used by runner and runner-ct in [this PR](https://github.com/cypress-io/cypress/pull/16866). So if that PR is merged before this one, you might need to rebase.

Reply:
FYI it's merged: https://github.com/cypress-io/cypress/pull/16866
thanks!

Reply:
Thanks for the update - that PR was desperately needed for a while now

Plus merging it into this PR was surprisingly painless :)